### ISSUE TITLE
Allow `<listBibl>` in `<abstract>`

### ISSUE THREAD

**Issue by cthomasdta** [TEIC/TEI#2292]
In the teiHeader of letters we give a summary of the relevant contents of said letter in `../profileDesc/abstract/p`, cf. `<abstract>` 
element, https://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-abstract.html. Additional to that `abstract/p` we keep lists of different entities mentioned, and would like to keep the lists *inside* the `<abstract>` element. For these lists, we happily use `<listPlace>`, `<listObject>`, `<listOrg>`, `<listPerson>` et al.

We would like to also list the works mentioned (books, plays, manuscripts etc., whatever fits into `<bibl>`) accordingly.
**But: `<listBibl>` is not allowed inside `<abstract>`. We would like to ask that this is changed.**

Summing up the short discussion on TEI-L, cf. https://listserv.brown.edu/cgi-bin/wa?A2=TEI-L;fc7e06ed.2204: 

Martin Holmes:
> Oddly enough, `<listBibl>` appears not to be a member of model.listLike:
> <https://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-model.listLike.html>
> which is why it's not available in `<abstract>`.

Bauman, Syd:
> Short answer: I think TEI should probably allow `<listBibl>` in `<abstract>`.

(*NB:* the following "BUT"-musings and my answer to it are not relevant to the issue at stake here.)

So, @martindholmes and @sydb have agreed, and unless there are no objections we have not foreseen yet, we would ask to allow `<listBibl>` in `<abstract>`.

**Comment by sabineseifert** [comment]
Direct approach to solving the issue could be adding `listBibl` to the content model of `abstract` like this:

```xml
<content>
 <alternate minOccurs="1" maxOccurs="unbounded">
  <classRef key="model.pLike"/>
  <classRef key="model.listLike"/>
  <elementRef key="listBibl"/>
 </alternate>
</content>
```

Making `listBibl` member of `model.listLike` seems like a can of worms...

**Comment by lb42** [comment]
This would of course permit an abstract containing nothing but a listBibl. Which makes me at least a little uncomfortable.

**Comment by peterstadler** [comment]
Well, it already does allow for only an e.g. `listPlace`. @lb42, would you feel more comfortable it at least one `model.pLike` was mandatory?
Any other ideas?

**Comment by martinascholger** [comment]
Council F2F agrees with @sabineseifert proposal
